ovgs.proto

```diff
@@ -230,7 +230,8 @@ message GetOwnershipVoucherResponse {
 service OwnershipVoucherService {
   // CreateGroup creates a group as a child of an existing group.
   // Errors will be returned:
-  // ALREADY_EXISTS if group already exists
```

Comment:
`ALREADY_EXISTS if group already exists` was removed because the group_id (which right now identifies a group) is server side generated.

We can make the pair (parent, description) unique and return ALREADY_EXISTS if that pair exists.